### ISSUE TITLE
We need a markdown linting tools for developers.

### ISSUE THREAD

**Issue by lijiarui** [wechaty/docusaurus#1392]
Juzibot have a markdown tool for non-developers to write markdown: https://meeken1998.github.io/chinese-document.html

I think we need a markdown tool as well for our community.

**Comment by Meeken1998** [comment]
I will try when im free (maybe

**Comment by livingbody** [comment]
typora is very beautifual.....
but building rules  is very strict...

**Comment by huan** [comment]
We have a vert simple solution that works out-of-the-box:

1. use vscode to open our wechaty.js.org repo
1. install the `markdownlint` vscode extension (it will use [this config file](https://github.com/wechaty/wechaty.js.org/blob/main/.markdownlint.json) automatically)
1. install the `editorconfig` vscode extension (it will use [this config file](https://github.com/wechaty/wechaty.js.org/blob/main/.editorconfig) automatically)
1. You are set. VSCode will tell you everything

@livingbody @Meeken1998

**Comment by lijiarui** [comment]
@huan You are right, but not all contributors have vscode, such as: https://github.com/wechaty/wechaty.js.org/pull/1115

**Comment by livingbody** [comment]
> 2\. markdownlint

what a good man you are! +999

**Comment by huan** [comment]
@livingbody You are welcome.

@lijiarui VSCode is not a big deal, today, it's just a web page.
